data/phi1038/phi001/__cts__.xml

```diff
@@ -1,8 +1,11 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1038.phi001" xml:lang="lat" groupUrn="urn:cts:latinLit:phi1038">
             <ti:title xml:lang="lat">Facta et Dicta Memorabilia</ti:title>
             <ti:edition urn="urn:cts:latinLit:phi1038.phi001.perseus-lat1" workUrn="urn:cts:latinLit:phi1038.phi001">
-              <ti:label xml:lang="eng">Facta et Dicta Memorabilia, Factorum et dictorum memorabilium libri novem cum Iulii Paridis et Ianuarii Nepotiani epitomis</ti:label>
-              <ti:description xml:lang="eng">Valerius Maximus, creator; Kempf, Karl Friedrich, 1819-, editor</ti:description>
+              <ti:label xml:lang="lat">Facta et Dicta Memorabilia</ti:label>
+              <ti:description xml:lang="mul">Valerius Maximus. Kempf, Karl Friedrich, editor. Factorum et 
+                dictorum memorabilium libri novem cum Iulii Paridis et Ianuarii Nepotiani epitomis. Leipzig: Teubner,
+                1888.
+              </ti:description>
             </ti:edition>
           </ti:work>
```

Comment:
See my last review. I think editor is out of order. Editor should follow the book title.

Reply:
I will fix this one as well! Thanks @lcerrato